source/javascripts/controllers/_WorkflowsController.js.erb

```diff
@@ -538,6 +538,13 @@ import semverDiff from 'semver/functions/diff'
           var wrappedUserStepConfig = {};
           wrappedUserStepConfig[aStep.cvs] = aStep.userStepConfig;
 
+          if (aStep.isWithBlock()) {
+            aStep.withBlockData = {
+              image: appService.getwithBlockImage(aStep.userStepConfig?.container),
+              services: appService.getwithBlockServices(aStep.userStepConfig?.services)
```

Comment:
```suggestion
              services: appService.getWithBlockServices(aStep.userStepConfig?.services)
```

Reply:
Thank you @zoltanszabo-bitrise for all of your comments, I've fixed our naming convention mistakes everywhere :)